web-common/src/features/dashboards/time-controls/TimeRangeSelector.svelte

```diff
@@ -138,14 +156,16 @@
 >
   {#if hasSubRangeSelected}
     <div class="flex">
-      <TimeRangeScrubChip
-        on:click={toggleFloatingElement}
-        on:remove={() => dispatch("remove-scrub")}
-        {active}
-        start={$dashboardStore?.selectedScrubRange?.start}
-        end={$dashboardStore?.selectedScrubRange?.end}
-        zone={$dashboardStore?.selectedTimezone}
-      />
+      {#if $dashboardStore?.selectedScrubRange?.start && $dashboardStore?.selectedScrubRange?.end}
```

Comment:
We can instead modify `hasSubRangeSelected` to be equal to `$dashboardStore?.selectedScrubRange?.start && $dashboardStore?.selectedScrubRange?.end` and remove this check

The purpose of just adding just `end` to `hasSubRangeSelected` before was that it was always implied that `start` would exist if `end` exists.

Reply:
hey @djbarnwal , I did it a bit differently, LMK what you think